src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -827,7 +826,7 @@ public async ValueTask DisposeAsync()
 }";
             var comp = CreateCompilationWithTasksExtensions(new[] { source, s_IAsyncEnumerable }, options: TestOptions.DebugExe);
             comp.VerifyDiagnostics();
-            CompileAndVerify(comp, expectedOutput: "Got(1) Got(2) Captured(1)", verify: Verification.Skipped);
```

Comment:
:memo: I could not figure out why I had to skip the verification in the first place. I feel I would have opened a follow-up issue with details, but could not find one. Anyways, there seems to be no problem removing now.